org.adempiere.base.process/src/org/compiere/process/RoleAccessUpdate.java

```diff
@@ -121,7 +126,7 @@ protected String doIt () throws Exception
 	 */
 	private void updateRole (MRole role)
 	{
-		StringBuilder msglog = new StringBuilder(role.getName()).append(": ") 
+		StringBuilder msglog = new StringBuilder(getAD_Client_ID() > 0 ? "" : MClient.get(role.getAD_Client_ID()).getName() + " - ").append(role.getName()).append(": ") 

```

Comment:
I think it is fine to remove the ad_client_id > 0 check and always show the tenant/client name.

Reply:
done